catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/FederationAdminServiceImpl.java

```diff
@@ -196,37 +197,38 @@ public void updateRegistryEntry(Metacard updateMetacard) throws FederationAdminE
     public void updateRegistryEntry(Metacard updateMetacard, Set<String> destinations)
             throws FederationAdminException {
         validateRegistryMetacards(Collections.singletonList(updateMetacard));
-        String registryId = RegistryUtility.getRegistryId(updateMetacard);
-        if (registryId == null){
-            return;
-        }
-
-        List<Metacard> existingMetacards =
-                this.getRegistryMetacardsByRegistryIds(Collections.singletonList(registryId));
 
-        if (existingMetacards.size() > 1) {
-            String message = "Error updating registry entry. Multiple registry metacards found.";
-            List<String> metacardIds = existingMetacards.stream()
-                    .map(Metacard::getId)
-                    .collect(Collectors.toList());
-            LOGGER.error("{} Matching registry metacard ids: {}", message, metacardIds);
+        Map<String, Serializable> properties = new HashMap<>();
+        String mcardId = updateMetacard.getId();
```

Comment:
Should we be check the metacard for null? It happened elsewhere for a null metacard on an update.